### DISCUSSION TITLE
Brewfest & hallows end 2024

### DISCUSSION THREAD

**Discussion by Kirishiki** [chromiecraft/chromiecraft#7295] [Suggestions]
i would request that chromiecraft would polish these events for this year to work 100%, and the suggestion to have the lvl 70 gear to drop from the bosses aswell. last year the brewfest music was bit off, as u wandered trough brewfest grounds the music kept shifting alot to actual zone music on 1 step and to the brewfest music on next step, and switch to different brewfest music on third so that needs bit of work. also there was bit of issue with the ram quest to keep specific speed up, it changed with bit odd way compared to what it should be.

**Comment by BeefSteaker** [comment]
The 70 items will likely be available, they were only removed as they'd have been bis the whole of tbc if they were available last year. 

I don't believe the music or ram quests have an open report, so if it's still the case then you'd need to open ones for any chance of that being fixed

**Reply by Kirishiki** [reply]
good to hear for the gear part, i made bug report for the music and ram quest.
